packages/desktop-client/src/components/modals/SelectLinkedAccountsModal.tsx

```diff
@@ -43,7 +74,7 @@ export function SelectLinkedAccountsModal({
   });
 
   async function onNext() {
-    const chosenLocalAccountIds = Object.values(chosenAccounts);
+    const chosenLocalAccountIds: string[] = Object.values(chosenAccounts);
```

Comment:
Can you type this using AccountEntity?

```suggestion
    const chosenLocalAccountIds: AccountEntity['id'][] = Object.values(chosenAccounts);
```